iiasa/message_ix
# Update docs for v3.3.0; remove rmessageix

This PR addresses several issues related to documentation, in preparation for v3.3.0.

One improvement: it adds TeX macros like e.g. `\STORAGECHARGE`, which MathJax expands to e.g. `\text{STORAGE_CHARGE}`—these are a little more readable than the default TeX treatment of every single letter (S, T, O, R, etc.) as a distinct symbol rendered in italics. The PR does not apply this improvement to existing equations.

Also:
- Remove rmessageix, per https://github.com/iiasa/message_ix/issues/412#issuecomment-712020897, and update install instructions.
- Re-enable CI builds on Python 3.9 (parallel to iiasa/ixmp#414).

## How to review

- Build the documentation and look at:
  - Page “Sets and mappings”, new section “Index names”.
  - Page “Install”, section on R.
  - Page “Usage from R via reticulate”.
  - Page “Model core formulation”, equations STORAGE_BALANCE_INIT and STORAGE_EQUIVALENCE.
- Check (per https://github.com/iiasa/message_ix/issues/457#issuecomment-822445020) that stacked bar charts appear reasonable in both Austria and Westeros tutorials.

## PR checklist
